go.mod

```diff
@@ -35,10 +34,10 @@ require (
 
 require (
 	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
-	github.com/DataDog/datadog-api-client-go/v2 v2.27.0 // indirect
+	github.com/DataDog/datadog-agent/pkg/obfuscate v0.59.0-rc.5 // indirect
+	github.com/DataDog/datadog-agent/pkg/remoteconfig/state v0.59.0-rc.5 // indirect
```

Comment:
do we need this datadog-agent package in WPA?

Reply:
These 2 are unrelated to my changes. They were added when I ran `go mod tidy`, so it was either out of sync or there's an issue in my setup.

Both of them are dependencies of `gopkg.in/DataDog/dd-trace-go.v1` (already there before my PR).

```
go mod why -m github.com/DataDog/datadog-agent/pkg/obfuscate
# github.com/DataDog/datadog-agent/pkg/obfuscate
github.com/DataDog/watermarkpodautoscaler
gopkg.in/DataDog/dd-trace-go.v1/profiler
gopkg.in/DataDog/dd-trace-go.v1/profiler.test
gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer
github.com/DataDog/datadog-agent/pkg/obfuscate
```

```
go mod why -m github.com/DataDog/datadog-agent/pkg/remoteconfig/state
# github.com/DataDog/datadog-agent/pkg/remoteconfig/state
github.com/DataDog/watermarkpodautoscaler
gopkg.in/DataDog/dd-trace-go.v1/profiler
gopkg.in/DataDog/dd-trace-go.v1/profiler.test
gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer
github.com/DataDog/datadog-agent/pkg/remoteconfig/state
```

In any cases these are specific packages in the agent so it shouldn't cause any issues when upgrading k8s libs.

Reply:
ok thanks for looking at it. 
I was scared that we have yet another cross project dependency.